sanic/blueprint_group.py

```diff
@@ -0,0 +1,93 @@
+class BlueprintGroup(object):
+    """
+    This class provides a mechanism to implement a Blueprint Group
+    using the `Blueprint.group` method. To avoid having to re-write
+    some of the existing implementation, this class provides a custom
+    iterator implementation that will let you use the object of this
+    class as a list/tuple inside the existing implementation.
+    """
+
+    def __init__(self, url_prefix=None):
+        """
+        Create a new Blueprint Group
+
+        :param url_prefix: URL: to be prefixed before all the Blueprint Prefix
+        """
+        self._blueprints = list()
```

Comment:
`self._blueprints = []`

Reply:
I usually prefer `list()` and `dict()` because of readability.  But if `[]` is the standard, then I can change it.